test/test_response.py

```diff
@@ -801,7 +800,7 @@ def test_invalid_chunk_length(self):
 
         orig_ex = ctx.value.args[1]
         assert isinstance(orig_ex, InvalidChunkLength)
-        assert orig_ex.length == six.b(fp.BAD_LENGTH_LINE)
+        assert orig_ex.length == fp.BAD_LENGTH_LINE.encode("latin-1")
```

Comment:
We should remove it and only do `.encode()` to not confuse future readers into thinking there's anything relevant about the "latin-1" encoding here.